src/issue-priority.html

```diff
@@ -1,27 +1,39 @@
 <link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/iron-icon/iron-icon.html">
+<link rel="import" href="../bower_components/iron-icons/image-icons.html">
 
 <dom-module id="issue-priority">
   <template>
     <style>
       :host {
-        display: block;
-
-        padding: 10px 20px;
+        display: flex;
+      }
+      :host .icon {
```

Comment:
As a general tip about using `:host` – it's usually used in only two cases. 1) targeting the host element itself (like you've done above) 2) applying styles based on the host attributes or classes like `:host(.foobar) .icon`. 

In this case, we're targeting the styles for a child element inside the shadow root without caring about any attributes or classes on the host element, so no `:host` is needed.